rules/internal/utils/method.go

```diff
@@ -36,6 +36,23 @@ func IsCreateMethod(m *desc.MethodDescriptor) bool {
 	return createMethodRegexp.MatchString(m.GetName())
 }
 
+// IsCreateMethod returns true if this is a AIP-133 Create method with
+// a non-nil response type. This method should be used for filtering in linter
+// rules which access the response type of the method, to avoid crashing due
+// to dereferencing a nil pointer to the response.
+func IsCreateMethodWithResolvedReturnType(m *desc.MethodDescriptor) bool {
+	if !IsCreateMethod(m) {
+		return false
+	}
+
+	res := m.GetOutputType()
+
+	if info := GetOperationInfo(m); info != nil {
+		res = FindMessage(m.GetFile(), info.GetResponseType())
+	}
+	return res != nil
```

Comment:
Could use [`GetResponseType`](https://pkg.go.dev/github.com/googleapis/api-linter/rules/internal/utils#GetResponseType) in this package:
```suggestion
	return GetResponseType(m) != nil
```